Projects/Chat/Sources/Models/Conversation.swift

```diff
@@ -12,6 +12,7 @@ public struct Conversation: Identifiable, Hashable, Codable {
         status: ConversationStatus,
         hasClaim: Bool,
         claimType: String?,
+        claimId: String?,
```

Comment:
I don't think we need this one. We will fetch it from the ChatViewModel. No need to have it in the conversation itself.